README.md

```diff
@@ -50,6 +50,33 @@ A simple library to generate and retrieve OAuth2 tokens for use with Google Clou
     }
     ```
 
+## Google Compute Metadata
+
+Every compute instance stores its metadata on a metadata server.
+Goth can query this metadata server to fetch authentication credentials
+for a service account within the instance.
+
+To query the metadata server for an access token, you must configure
+the following:
+
+  * `url` must be set to the base url for the metadata server.
+  * `credentials` must be set to a tuple `{:instance, String.t()}`
+    with the name of the service account to use.
+
+```elixir
+defmodule MyApp.Application do
+  use Application
+
+  def start(_type, _args) do
+    children = [
+      {Goth, name: MyApp.Goth, url: "http://metadata.google.internal", credentials: {:instance, "default"}}
```

Comment:
double-checking `"http://metadata.google.internal"` or `"https://..."`?

Reply:
I can confirm it's HTTP -- probably because it's internal
Here's a reference in the docs: https://cloud.google.com/appengine/docs/standard/java/accessing-instance-metadata#identifying_which_metadata_endpoint_to_use